forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -407,23 +386,30 @@ def delete(mapper_id: int):
     )
     def put(mapper_id: int):
         """Update form by mapper_id."""
-        application_json = request.get_json()
-
-        if "taskVariable" in application_json:
+        data = request.get_json()
+        mapper_data = data.get("mapperData")
```

Comment:
Can we change the keys to match resources ?
- mapper
- authorizations
- taskVariables

Reply:
okay